CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# 4.4.0 / 2022-10-25
+
+### New Features
+
+- Featuring a new borderless design
+- Modified comments are now easily identifiable
+- Code preview on advanced search
+- The user interface is now available in Norwegian 
+- Video player for video previews
+- Improved pertinence of notifications about tasks
+- UX / UI improvements (sidebar, header, news, mobile)
+
+### Fixed Issues
+
+- Frontend: [#4600](https://github.com/tracim/tracim/issues/4600),
+  [#5510](https://github.com/tracim/tracim/issues/5510),
+  [#5526](https://github.com/tracim/tracim/issues/5526),
+  [#5625](https://github.com/tracim/tracim/issues/5625),
+  [#5747](https://github.com/tracim/tracim/issues/5747),
+  [#5748](https://github.com/tracim/tracim/issues/5748),
+  [#5760](https://github.com/tracim/tracim/issues/5760),
+  [#5858](https://github.com/tracim/tracim/issues/5858)
+- Backend: [#5526](https://github.com/tracim/tracim/issues/5526),
+  [#5844](https://github.com/tracim/tracim/issues/5844),
+  [#5878](https://github.com/tracim/tracim/issues/5878)
+
+### Breaking/Important change
+
+- :warning: Two new colors (`sidebar/font` and `sidebar/logo`) were added in `branding/color.json`: this requires all existing instances to update their `frontend/dist/assets/branding/color.json` files to add these new colors. Otherwise the backend won't start at all. For more details see: [#5860](https://github.com/tracim/tracim/issues/5860).
+- Norwegian has been added as a possible interface language. If you have Custom Properties specific to your instance, you will need to make specific translations as well. See the documentation at `backend/doc/user_custom_properties.md`
+- A new variable was added `ui.notes.code_sample_languages` (`TRACIM_UI__NOTES__CODE_SAMPLE_LANGUAGES`) that defines the list of languages available to create code samples in the notes.
+- Some API URLs are deprecated. See [#5860](https://github.com/tracim/tracim/issues/5860) for more details.
+
+### Known Issues
+
+- The positioning of the notification wall is to be improved (issue [#5965](https://github.com/tracim/tracim/issues/5965))
+- The "My profile" button does not change its name when viewing as an admin on another user's profile (issue [#5975](https://github.com/tracim/tracim/issues/5975))
```

Comment:
The "My profile" button does not change its name when using admin rights to check the profile of an other user.